corehq/apps/integration/payments/services.py

```diff
@@ -167,3 +169,15 @@ def _payment_is_verified(case_data: dict):
 
 def _payment_already_submitted(case_data: dict):
     return case_data.get(PaymentProperties.PAYMENT_SUBMITTED) == 'True'
+
+
+def get_payment_batch_numbers_for_domain(domain):
+    case_ids = (
+        CaseSearchES()
+        .domain(domain)
+        .case_type(MOMO_PAYMENT_CASE_TYPE)
+        .values_list("_id", flat=True)
+    )
```

Comment:
Nit: You could also simply use `get_ids()` here:
```suggestion
    ).get_ids()
```